test/config/integration_tests.yml

```diff
@@ -135,17 +135,6 @@ testCases:
   validator:
     class: ExportValidator 
 
-- name: nnicli
```

Comment:
why remove all the tests?

Reply:
I used to think that it is difficult to split the test to `launchCommand` and `stopCommand` since we use an instance to handle an experiment. Maybe I could connect to an existing experiment in `stopCommand`. Will add the tests again.